src/classlibnative/bcltype/number.h

```diff
@@ -1,47 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-//
-// File: Number.h
-//
-
```

Comment:
You can delete `// needs to match layout of NUMBER in coreclr's src/classlibnative/bcltype/number.h` command on the managed NumberBuffer since it is not applicable anymore.

Reply:
Removed.